stable/prometheus-postgres-exporter/values.yaml

```diff
@@ -75,8 +75,10 @@ config:
     database: ''
     sslmode: disable
   disableDefaultMetrics: false
+  disableSettingstMetrics: false
```

Comment:
Hey @jleloup,  is this supposed to be [disableSettingsMetrics](https://github.com/helm/charts/blob/b55555504e124c08535c02ffa2eed05dd17f6100/stable/prometheus-postgres-exporter/templates/deployment.yaml#L42) vs. `disableSettingstMetrics`? Seeing `disableSettingstMetrics` in the README table as well.

Reply:
This is definitely a typo. I'm gonna fix this